src/main/g8/default.properties

```diff
@@ -10,6 +10,10 @@ package = $organization$.$name;format="norm,word"$
 
 scala_version = 2.12.6
 sbt_version = 1.2.1
-http4s_version = 0.19.0-SNAPSHOT
+http4s_version = 0.19.0-M3
+scala_version = 2.12.7
+sbt_version = 1.2.3
+http4s_version = 0.19.0-M3
```

Comment:
We've got a duplicate version in here.  Seems to not hurt anything.  We can take it out when we bump to M4.